CHANGELOG.md

```diff
@@ -6,6 +6,41 @@ This file is generated automatically. Add individual changelog entries to the 'c
 
 This changelog documents all notable changes to VAST and is updated on every release. Changes made since the last release are in the [changelog/unreleased directory][unreleased].
 
+## [v1.1.0-rc1]
+
+### :warning: Changes
+
+- VAST no longer attempts to intepret query expressions as Sigma rules automatically. Instead, this functionality moved to a dedicated `sigma` query language plugin that must explicitly enabled at build time.
+  [#2074](https://github.com/tenzir/vast/pull/2074)
+
+- The `msgpack` encoding option is now deprecated. VAST issues a warning on startup and automatically uses the `arrow` encoding instead. A future version of VAST will remove this option entirely.
+  [#2087](https://github.com/tenzir/vast/pull/2087)
+
+- The experimental aging feature is now deprecated. If you require this feature, please contact us about the [compaction plugin](https://docs.tenzir.com/vast/features/compaction) which is its more capable replacement.
+  [#2087](https://github.com/tenzir/vast/pull/2087)
+
+### :gift: Features
+
+- The built-in 'select' transform step may now use filter expressions that match *all* events of the input.
+  [#2064](https://github.com/tenzir/vast/pull/2064)
+
+- VAST has a new `query_language` plugin type that allows for adding additional query language frontends. The plugin performs one function: compile a string into a VAST expression. The new `sigma` plugin demonstrates usage of this plugin type.
```

Comment:
```suggestion
- VAST has a new *query language plugin* that allows for adding additional query language frontends. The plugin performs one function: compile user input into a VAST expression. The new `sigma` plugin demonstrates usage of this plugin type.
```

Reply:
This is wrong. There's an abstraction for query language plugins, but that does not mean that there is a query language plugin.

Reply:
I took your suggestion, but left in the word type.

Reply:
Makes sense, actually.